### DISCUSSION TITLE
Add a shortcut for previous and next script in script editing history

### DISCUSSION THREAD

**Discussion by psychicteeth** [godotengine/godot-proposals#11158] [Editor]
TL;DR: It would be great to have a shortcut that cycles back (and forward) through recently edited scripts.

At the moment, we have shortcuts for previous and next script that just move the focus up or down in the alphabetised list of open scripts. It would be far more useful to be able to skip forward and backward through recently edited scripts. I have a bunch of scripts I edit and frequently need to jump back and forth between recently edited ones, and currently there's no way to do that besides stopping what I'm doing, looking at the list, finding the one I want and clicking on it, or hammering next/previous script until I come across it.

Most code IDEs have this feature bound to ctrl-tab and ctrl-shift-tab by default and I think many coders probably rely on it for efficiency. Godot binds these shortcuts to previous/next scene, which DOES work with history. Personally I would consider this a bug insofar as the functionality between two similarly named features is different (with scenes, it uses history; with scripts, it uses alphabetised list). However I'm sure people are used to the existing shortcuts so I'm submitting this as a new feature proposal.

**Comment by tetrapod00** [comment]
See previous discussion here: https://github.com/godotengine/godot/issues/97031. I believe `alt+left` and `alt+right` previously did this, then were adjusted to navigate to finer increments of last locations *within* a file.
